pkg/podnetwork/tunneler/vxlan/workernode.go

```diff
@@ -18,8 +18,8 @@ import (
 var logger = log.New(log.Writer(), "[tunneler/vxlan] ", log.LstdFlags|log.Lmsgprefix)
 
 const (
-	vxlanPort                = 4789
-	vxlanMinID               = 555000
+	DefaultVXLANPort         = 4789
```

Comment:
Should we change default port to another one? so that it's not conflict with calico without passing VxLAN port? I know 4789 is well known, that's why it's possible to conflict.

Reply:
In this case, Calico behaves wrongly. The VXLAN port is 4789, which is defined in RFC, and we can distinguish VXLAN packets by VXLAN ID (VNI).

Calico should not drop VXLAN packets that have VXLAN IDs other than 4096, which is the VXLAN ID for packets encapsulated by Calico.

I will raise an issue at Calico's GitHub repo.